src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -68,6 +69,8 @@ public static int MainCore(string workingDirectory, string[] args)
             }
 
             var console = new Common.Console();
+            Logger.Instance = console;
```

Comment:
Are we and should we call `Program.Main` directly in parallel? What commandline scenarios are we testing that way? Each run has its own console output, the instance is created on line 70.